web/admin.py

```
@@ -812,3 +821,141 @@ class PointsAdmin(admin.ModelAdmin):
+    actions = ["approve_selected", "reject_selected"]
+    
+    def approve_selected(self, request: HttpRequest, queryset: QuerySet["ChapterApplication"]) -> None:
+        pending = queryset.filter(status='pending')
+        for application in pending:
+            application.status = 'approved'
+            application.processed_by = request.user
+            application.save()
+        self.message_user(request, f"{pending.count()} applications approved.")
+    approve_selected.short_description = "Approve selected applications"
+    
+    def reject_selected(self, request: HttpRequest, queryset: QuerySet["ChapterApplication"]) -> None:
```

Ruff (0.8.2)

Undefined name `HttpRequest`

(F821)